### DISCUSSION TITLE
Question about telegram install information

### DISCUSSION THREAD

**Discussion by dmhjaques** [python-telegram-bot/python-telegram-bot#2906] [Q&A] [answered]
Dear All,
  
The following are questions about telegram software. This question is very important and will have a very huge impact on some people's lives. 
The problem is about an Android phone (HuaWei) and the critical issue is whether telegram ever installed in this phone. Evidence shows there are no information about telegram in the log of this phone. But there is an telegram icon in the res floder of Root directory.
So, to put it simply, we want to know: 1. During the period from 2016 to 2017, if telegram is deleted after installing, will the installed information logged in the log of android of android operation system? 2. If telegram was installed and has been deleted, is it possible that the PNG icon of the telegram software will be left in the res folder in Root directory? 3. If the answer to the first two questions is no, why the icon of Telegram appears in the res folder?
  
 If anyone knows, please contact me.

**Comment by Bibo-Joshi** [comment, marked as answer]
This Q&A forum is concerned with the `python-telegram-bot` library, a Python library for developing Telegram __bots__. It's not in any way connected with Telegram clients that are installed on mobile devices.
